tools/render-test/render-d3d11.cpp

```diff
@@ -166,19 +160,19 @@ Renderer* createD3D11Renderer()
     // shader bytecode as part of an offline process, rather than doing it
     // on-the-fly like this
     //
-    static pD3DCompile D3DCompile_ = nullptr;
-    if (!D3DCompile_)
+    static pD3DCompile compileFunc = nullptr;
```

Comment:
More unstated convention stuff that fits in the "I don't have a strong opinion" pile: usually if I'm dynamically loading a function from a shared library, I make my variable that represents the function either match the name of the symbol (e.g., for the GL/VK procs we load) or else something obvious derived from it (the `_` suffix in this case).